examples/material_gallery/lib/main.dart

```diff
@@ -46,14 +46,21 @@ class GallerySection extends StatelessComponent {
       brightness: ThemeBrightness.light,
       primarySwatch: colors
     );
+    final appBarHeight = 200.0;
```

Comment:
Yes, I think you're correct. The figure on the right side of the "Detail View" section of 
https://www.google.com/design/spec/layout/metrics-keylines.html#metrics-keylines-keylines-spacing says that the size of the area below the toolbar/statusbar should have an aspect ration of 3:2.  The current API really needs a fixed appBar height. I'll file an issue for this.